Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -235,6 +228,8 @@ @interface ConversationViewController () <AttachmentApprovalViewControllerDelega
 @property (nonatomic) BOOL viewHasEverAppeared;
 @property (nonatomic) BOOL hasUnreadMessages;
 @property (nonatomic) BOOL isPickingMediaAsDocument;
+@property (nonatomic, nullable) NSNumber *previousLastTimestamp;
+@property (nonatomic, nullable) NSNumber *previousLastUnreadTimestamp;
```

Comment:
We now do a much better job updating the view when we resume observing database modifications.